evaluator/evaluate_packages.go

```diff
@@ -252,6 +252,9 @@ func vmaasResponse2UpdateDataJSON(updateData *vmaas.UpdatesV2ResponseUpdateList)
 			uniqUpdates[pkgUpdate] = true
 		}
```

Comment:
If we want to store only latest update it's inefficient to create whole array, make it unique and then take only last item from it :).

What about
```
         if enablePrunePackageLatestOnly {
			pkgUpdates[0] = pkgUpdate
         } else {
                if !uniqUpdates[pkgUpdate] {
			pkgUpdates = append(pkgUpdates, pkgUpdate)
			uniqUpdates[pkgUpdate] = true
		}
         }
```

Reply:
We need the last item, not the first one, And in the case of an invalid data received we would have to iterate through the array from the end. Current code is the simplest approach to get our desired result, and performance of the evaluator isn't remotely the problem we're facing.

Reply:
`pkgUpdates[0] = pkgUpdate` rewrites the only value in array/slice so in the end there the last valid value.

Reply:
@semtexzv can you pls, answer that?